device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/gearbox_config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "phys": [
```

Comment:
where can i find the the format for this gearbox_config.json?

Reply:
https://github.com/Azure/SONiC/blob/17f89b8e415f75b8bc4fad5fc9d6c46fb132a86c/doc/gearbox/gearbox_mgr_design.md

Reply:
https://github.com/Azure/SONiC/blob/b817a12fd89520d3fd26bbc5897487928e7f6de7/doc/gearbox/gearbox_mgr_design.md